### PR TITLE
Cloud Native Chapter 5

### PR DESCRIPTION
**Pull request by ardalis**
## Summary

Chapter 5 files. TOC and images are in ch1 PR.

### CODE REVIEW

File: docs/architecture/cloud-native/data-patterns.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+NoSQL databases can be categorized by one of the following four models: 
+
+-   *Document Store* (mongodb, couchdb, couchbase) - Data (and corresponding metadata) is stored non-relationally in denormalized JSON-based documents inside the database
+
+-   *Key/Value Store* (redis, riak, memcached) � Data is stored in simple key-value pairs with system operations performed against a unique access key that is mapped to a value of user data
```
Comment (mairaw) [inline docs/architecture/cloud-native/data-patterns.md]: can you fix these characters @ardalis?

**Comment by mairaw** [comment]
There's still some left apparently

**Comment by ardalis** [comment]
looks good now

**Comment by mairaw** [comment]
Ok, thanks @ardalis. @rpetrusha volunteered to review this chapter.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
I've made some editorial and formatting corrections for you to review, @ardalis. In addition, the Cosmos DB and Azure Storage Services sections in the data storage article need to be completed.

**Comment by ardalis** [comment]
@robvet do you want to complete Cosmos and Azure Storage Services in this PR or as a follow-on?

**Comment by robvet** [comment]
It’s been done. I’m updating that md right now.


Thanks!

Rob Vettor | Principal Cloud System Architect
                      | ASD IP Architect
                      | Developer Advisory Services for Azure

[Description: cid:image001.gif@01CC9CCD.D94B9620]  214-707-0584
[Description: cid:image003.gif@01CC9CCD.D94B9620]   robvet@microsoft.com<mailto:robvet@microsoft.com>
[cid:image003.jpg@01D52A88.9FF8CFE0]  www.thinkingincloudnative.com<http://www.thinkingincloudnative.com/>

**Comment by robvet** [comment]
Give me an hour to finish updating it and I’ll do a PR.

Thanks!

Rob Vettor | Principal Cloud System Architect
                      | ASD IP Architect
                      | Developer Advisory Services for Azure

[Description: cid:image001.gif@01CC9CCD.D94B9620]  214-707-0584
[Description: cid:image003.gif@01CC9CCD.D94B9620]   robvet@microsoft.com<mailto:robvet@microsoft.com>
[cid:image004.jpg@01D52A88.EFC67FE0]  www.thinkingincloudnative.com<http://www.thinkingincloudnative.com/>

From: Robert Vettor
Sent: Monday, June 24, 2019 12:31 PM
To: dotnet/docs <reply@reply.github.com>; dotnet/docs <docs@noreply.github.com>
Cc: Mention <mention@noreply.github.com>
Subject: RE: [dotnet/docs] Cloud Native Chapter 5 (#12932)

It’s been done. I’m updating that md right now.


Thanks!

Rob Vettor | Principal Cloud System Architect
                      | ASD IP Architect
                      | Developer Advisory Services for Azure

[Description: cid:image001.gif@01CC9CCD.D94B9620]  214-707-0584
[Description: cid:image003.gif@01CC9CCD.D94B9620]   robvet@microsoft.com<mailto:robvet@microsoft.com>
[cid:image005.jpg@01D52A88.EFC67FE0]  www.thinkingincloudnative.com<http://www.thinkingincloudnative.com/>

**Comment by ardalis** [comment]
@rpetrusha Are we good to merge now?

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
I've approved and I'll merge now, @ardalis.

**Pull request merged**